scheduler/cloudformation/template.go

```diff
@@ -235,6 +235,13 @@ func (t *EmpireTemplate) addTaskDefinition(tmpl *troposphere.Template, app *sche
 	cd := t.ContainerDefinition(app, p)
 	containerDefinition := cloudformationContainerDefinition(cd)
 
+	// If provided in the app environment, this role will be used when
+	// running tasks.
+	var taskRole interface{}
+	if arn, ok := app.Env["TASK_ROLE_ARN"]; ok {
```

Comment:
Yeah, there's a couple of other places where we decide how to build the template based on environment variables, and it would probably be better if we prefixed those with something (other ones are `ECS_TASK_DEFINITION` and `LOAD_BALANCER_TYPE`).

I'm game for deprecating those, and switching them to `EMPIRE_*` prefixed.